api/src/board.ts

```diff
@@ -13,24 +12,6 @@ enum Mode {
   custom = "2",
 }
 
-export async function getBoard(mode: string): Promise<Board> {
-  if (mode === Mode.initial) {
-    return getInitialBoard();
-  }
-
-  const boardPath = await askUserBoardPath();
-  return getBoardFromFile(boardPath);
-}
-
-export function getBoardFromFile(customPath: string): Board {
-  try {
-    const data = readFileSync(customPath, { encoding: "utf8" });
-    return JSON.parse(data).board;
-  } catch (err) {
-    return [];
-  }
-}
-
```

Comment:
Why did you remove those ?

Reply:
I modified the type, but not the fetch inside because we modify it in the next PR